src/components/Hoverable/index.js

```diff
@@ -81,7 +81,7 @@ class Hoverable extends Component {
                     }
                 },
                 onBlur: (el) => {
-                    if (!this.wrapperView.contains(el.relatedTarget)) {
+                    if (this.wrapperView && !this.wrapperView.contains(el.relatedTarget)) {
                         this.setIsHovered(false);
```

Comment:
Again why we are handling this here?

Reply:
@Santhosh-Sellavel Sorry, just to match the below, I have removed it